src/scripts/items/VitaminController.ts

```diff
@@ -0,0 +1,13 @@
+class VitaminController {
+    public static counter = 0;
```

Comment:
What is the counter being used for?

Reply:
overlooked removing it. done, thanks for catching it :D